sass/components/_barrier.scss

```diff
@@ -87,13 +89,13 @@
 }
 
 .drag-price {
-    display: block;
-    height: 23px;
-    top: -11px;
+    display: flex;
+    height: 24px;
```

Comment:
Just asking , we don't follow rem instead of px in smartcharts?

Reply:
No, in deriv-app rem is fixed to 10px, while in smartcharts it's not, so it would be given a dynamic value which various on different devices, which we don't want;